root/static/scripts/common/components/Autocomplete2.js

```diff
@@ -434,18 +437,18 @@ export default function Autocomplete2<+T: EntityItem>(
       return;
     } else if (
       items.length > 0 &&
-      cleanInputValue === clean(props.inputValue)
+      cleanInputValue === clean(inputValue)
     ) {
       dispatch(SHOW_MENU);
     } else if (nonEmpty(cleanInputValue)) {
       if (staticItems) {
         dispatch({
-          searchTerm: inputValue,
+          searchTerm: newInputValue,
```

Comment:
Is this meant to be new rather than clean?

Reply:
I don't remember but it looks like this code got removed or rewritten in a later commit.

Reply:
Since it's filtering, it probably did intend to use `newInputValue` though since spaces affect that.